### ISSUE TITLE
GD-89: Add a option to install examples

### ISSUE THREAD

**Issue by MikeSchulze** [MikeSchulze/gdUnit3#89]
**Is your feature request related to a problem? Please describe.**
With GD-88 we exclude all integration and examples form AssetLib download.
It would be nice to have a option to install examples to can look into and learn how to write tests.

**Describe the solution you'd like**
Add an option like a button to optional install GdUnit examples.

**Describe alternatives you've considered**
manual download from GitHub

**Comment by clemens-tolboom** [comment]
In #88  the examples are not excluded as `gdUnit3-examples` is not excluded but `examples`

FWIW AssetLib users have the opportunity to deselect download parts which IMHO === 'this issue #89' solution :-)

**Comment by MikeSchulze** [comment]
Yes the renaming was after #88 was finished.
The current version has the right exclude have a look at https://github.com/MikeSchulze/gdUnit3/blob/master/.gitattributes

**Comment by clemens-tolboom** [comment]
My point was to remove
```
/gdUnit3-examples
```
from `.gitattributes` thus fixing this issue ... AssetLib users can **deselect** the `/gdUnit-examples`

**Comment by clemens-tolboom** [comment]
Regarding

> In #88 the examples are not excluded as gdUnit3-examples is not excluded but examples

I don't know what went wrong ... I did see `/examples` :-/ ... let's call it a github glitch :-)

**Comment by MikeSchulze** [comment]
I have checked the AssetLib and still holds the v0.9.4, it is not updated.
My change request is not accepted yet, you have to wait ;)

**Comment by MikeSchulze** [comment]
The AssetLib is now updated and version v0.9.6 is available
